CHANGELOG.md

```diff
@@ -1,12 +1,24 @@
 CHANGELOG
 =========
 
+* fixed the `AllowedMethodsRouteLoader` to work with Symfony 3.0
+
 1.7.8
 -----
 
+* remove uses of the reflection API in favor of faster solutions when possible
+
+* fixed the configuration to use serialization groups and versions at the same time
+
 1.7.7
 -----
 
+* do not call methods on Symfony 3.x installations that have been removed in Symfony 3.0
```

Comment:
I don't understand this phrase. Does it mean: *"do not call methods that have been removed in Symfony 3"* ?